docs/write-custom-pipeline.md

```diff
@@ -0,0 +1,159 @@
+# Writing your own pipelines
+
+This tutorial will walk you through how to set up your first custom pipeline.
```

Comment:
Suggestion:
This tutorial will walk you through how to set up your first custom trainings pipeline.

Reply:
done